test/scripts/jenkins_selenium.sh

```diff
@@ -3,4 +3,6 @@
 set -e
 source "$(dirname $0)/_jenkins_setup.sh"
 
+"$(npm bin)/grunt" build --release;
```

Comment:
Is this ok to do? Do we care that we're not running tests against the SNAPSHOT build?